modules/ROOT/pages/manage-monitor-applications.adoc

```diff
@@ -1,106 +1,11 @@
-
-== Metrics
-
-Each Mule application and API gateway replica runs a metrics agent that collects and transmits metrics data to 
-Anypoint Monitoring via secure mutual TLS encryption. Each replica can cache up to 100 megabytes of metrics, after which the metrics data is rotated to provide storage for more recent data. In the event that metrics cannot be sent to Anypoint Monitoring, this typically provides at least 4 days worth of storage per replica. Note that custom metrics might consume more storage.
-
-[NOTE]
---
-Application monitoring metrics are not restored after:
-
-* A backup and restore of Runtime Fabric
-* An upgrade of Runtime Fabric
-* OS patching of Runtime Fabric nodes
---
-
-== Logs 
-
-Anypoint Monitoring provides access to xref:monitoring::logs.adoc[log data] for applications deployed to Rutime Fabric. To access logs with Anypoint Monitoring, you need a https://www.mulesoft.com/anypoint-pricing[Titanium subscription]. 
-
-Each Mule application and API gateway replica can store up to 450 megabytes of log data on disk, after which log data 
-is rotated to provide storage for more recent log data.
-
-Logs generated by each replica of a Mule application and API gateway are collected and written to disk on the worker 
-node on which they are running. Each worker node runs a single log forwarding agent, which monitors this location 
-and transmits the log data to Anypoint Monitoring.
-
-[NOTE]
-====
-The log forwarding agent can transmit up to 300 kilobytes of log data per second. You should not deploy more than
-50 replicas per worker node.
-====
-
-You can enable log forwarding and set log levels when you xref:deploy-to-runtime-fabric.adoc#configure-logging-and-log-levels[deploy applications in Runtime Manager]. 
-
-=== Anypoint Monitoring Log Forwarding Prerequisites
-
-The daemon-set that forwards logs to Anypoint Monitoring expects application containers to write logs to the following locations:
-
-* `/var/log/pods`
-* `/var/log/containers`
-* `/var/lib/docker/containers`
-
-The daemon-set also creates and writes on the host path `/var/lib/filebeat-data`. For log forwarding to work as expected, ensure that:
-
-* The Docker daemon config includes the `json-file` logging driver.
-* The Anypoint Monitoring log forwarder daemon-set container has write access to the host on the host path `/var/lib/filebeat-data`.
-
-== Anypoint Visualizer
-
-xref:visualizer::index.adoc[Anypoint Visualizer] displays views of different aspects of an application network graph. 
-You can use the graph to explore your application network, identify problems, and make decisions.
+By default, Runtime Fabric sends application metrics to Anypoint Monitoring. Titanium-plan customers also have application logs automatically sent to Anypoint Monitoring by default. For metrics and logs to be sent to Anypoint Monitoring, outbound network connectivity must be available. Refer to xref:use-anypoint-monitoring.adoc[].
```

Comment:
Suggestion:
For metrics and logs to be sent to Anypoint Monitoring, outbound network connectivity must be available, as described in xref:use-anypoint-monitoring.adoc[]